x/ibc-hooks/contracts/crosschain-swaps/src/ibc.rs

```diff
@@ -0,0 +1,56 @@
+use osmosis_std_derive::CosmwasmExt;
+
+#[derive(
+    Clone,
+    PartialEq,
+    Eq,
+    ::prost::Message,
+    serde::Serialize,
+    serde::Deserialize,
+    schemars::JsonSchema,
+)]
+pub struct Height {
```

Comment:
Is it important this struct be called `Height`, or can it be called `IbcCounterpartyHeight` ?

(Its confusing as its not just the one u64 as you'd expect)

Reply:
oh is this derived from proto tooling?

If so we should add a `AUTO-GENERATED` comment at top of file

Reply:
This is not auto generated. Wrote it manually cause osmosis-std didn't implemented. I think we have it now. I'll check and may replace this.

Reply:
Double checked. We don't have them yet, but there's an open issue: https://github.com/osmosis-labs/osmosis-rust/issues/52